### DISCUSSION TITLE
NomadNet GUI App Concept

### DISCUSSION THREAD

**Discussion by LinuxinaBit** [markqvist/Reticulum#420] [Ideas]
I made a concept for a potential GUI that's basically just a stripped down Firefox UI, but it seems to work surprisingly well.

![Screenshot_2023-12-14_22-29-02](https://github.com/markqvist/NomadNet/assets/105567407/6e60b41a-0778-4dbf-a0e0-bf6b18623dad)
![Screenshot_2023-12-14_22-33-26](https://github.com/markqvist/NomadNet/assets/105567407/076d9ae7-fe52-410f-9c37-afe3ea839c6b)

The hamburger menu in the top right and the right click menu would be the replacement for the keyboard shortcut bar a the bottom of the screen (keyboard shortcuts and navigation could still work fine as well).

Hope this at least gives you some inspiration if you ever decide to make a NomadNet GUI.

**Comment by LinuxinaBit** [comment]
Maybe a Thunderbird-esque design would work better :thinking:

**Comment by markqvist** [comment]
Very nice, this is looking great!

At least for any foreseeable future, I will not be the one working on actually implementing it though, since I have to prioritize other parts, but I hope somebody else will take it up! Would be very cool to see something like this!

**Comment by markqvist** [comment]
Moved to discussions for greater visibility.

**Comment by LinuxinaBit** [comment]
Because Reticulum MeshChat (https://github.com/liamcottle/reticulum-meshchat) exists, I think this is pretty much solved :tada: 

![image](https://github.com/user-attachments/assets/6f167c68-6f46-48b9-887a-b49a25591a80)
